### ISSUE TITLE
running on custom dataset

### ISSUE THREAD

**Issue by andreaceruti** [wjf5203/SeqFormer#15]
I would like to get explanations on how the algorithm could be run on a dataset which has labels different from your training dataset

**Comment by wjf5203** [comment]
Thanks for your attention!
You can convert your labels to the same format as YouTubeVIS, or modify the dataloader function in datasets/ytvos.py to fit your format.
